Makefile

```diff
@@ -101,7 +101,7 @@ boostci: $(BUILD_DEPS)
 .PHONY: boostci
 
 react: check-node-lts
-	npm install --no-audit --prefix react
+	npm install --no-audit --legacy-peer-deps --prefix react
```

Comment:
I _think_ this is safe on older versions of npm, but there is the option of just adding a `npm_config_legacy_peer_deps=yes` before `npm` to do it via environment variable which will be safely ignored if it's not supported.